xsd/siri_model/siri_journey-v2.0.xsd

```diff
@@ -828,26 +880,103 @@ To avoid this, the producer should transmit xxxPredictionUnknown instead of any
+   <xsd:element name="ActualBoardingPositionRef" type="BoardingPositionRefStructure" minOccurs="0">
+    <xsd:annotation>
+     <xsd:documentation>Actually recorded BOARDING POSITION. Can be used to indicate the actually used boarding position.</xsd:documentation>
+    </xsd:annotation>
+   </xsd:element>
+   <xsd:element name="ActualBoardingPositionName" type="NaturalLanguageStringStructure" minOccurs="0" maxOccurs="unbounded">
+    <xsd:annotation>
+     <xsd:documentation>Recorded BOARDING POSITION name. Can be used to indicate the actually used boarding position.</xsd:documentation>
+    </xsd:annotation>
+   </xsd:element>
+  </xsd:sequence>
+ </xsd:group>
+ <!-- ======================================================================= -->
+ <xsd:complexType name="StopAssignmentStructure">
+  <xsd:annotation>
+   <xsd:documentation>Type for assignment of a SCHEDULED STOP POINT to a physical location, in particular to a QUAY or BOARDING POSITION. +SIRI v2.0.</xsd:documentation>
+  </xsd:annotation>
+  <xsd:sequence>
+   <xsd:choice>
+    <xsd:group ref="QuayAssignmentGroup"/>
+	<xsd:group ref="BoardingPositionAssignmentGroup"/>
+   </xsd:choice>
+   <xsd:group ref="FlexibleStopLocationGroup"/>
   </xsd:sequence>
  </xsd:complexType>
+ <!-- ======================================================================= -->
+ <xsd:complexType name="PlannedStopAssignmentStructure">
+  <xsd:annotation>
+   <xsd:documentation>Type for assignment of a SCHEDULED STOP POINT to a physical location, in particular to a QUAY or BOARDING POSITION, according to the planned timetable. +SIRI v2.0.</xsd:documentation>
+  </xsd:annotation>
+  <xsd:sequence>
+   <xsd:choice>
+    <xsd:sequence>
+     <xsd:element name="AimedQuayRef" type="QuayRefStructure" minOccurs="0">
+      <xsd:annotation>
+       <xsd:documentation>Physical QUAY to use according to the planned timetable. +SIRI v2.0</xsd:documentation>
+      </xsd:annotation>
+     </xsd:element>
+     <xsd:element name="AimedQuayName" type="NaturalLanguageStringStructure" minOccurs="0" maxOccurs="unbounded">
+      <xsd:annotation>
+       <xsd:documentation>Scheduled Platform name. +SIRI v2.0</xsd:documentation>
+      </xsd:annotation>
+     </xsd:element>
+     <xsd:element ref="QuayType" minOccurs="0"/>
+	</xsd:sequence>
+	<xsd:sequence>
+     <xsd:element name="AimedBoardingPositionRef" type="BoardingPositionRefStructure" minOccurs="0">
+      <xsd:annotation>
+       <xsd:documentation>Physical BOARDING POSITION to use according to the planned timetable. +SIRI v2.1</xsd:documentation>
+      </xsd:annotation>
+     </xsd:element>
+     <xsd:element name="AimedBoardingPositionName" type="NaturalLanguageStringStructure" minOccurs="0" maxOccurs="unbounded">
+      <xsd:annotation>
+       <xsd:documentation>Scheduled BOARDING POSITION name. +SIRI v2.1</xsd:documentation>
+      </xsd:annotation>
+     </xsd:element>
+	</xsd:sequence>
+   </xsd:choice>
+   <xsd:element ref="AimedFlexibleArea" minOccurs="0"/>
```

Comment:
Is FlexibleArea ADDITIONAL to Quay/ BoardingPosition or is it an ALTERNATIVE to those two? In the latter case it should be inside the 'choice'.

Reply:
FlexibleStopLocation was meant as ADDITIONAL. Does that work for you?